.travis.yml

```diff
@@ -25,6 +25,12 @@ install:
       mkdir -p $HOME/gopath/src/github.com/google
       ln -s $TRAVIS_BUILD_DIR $HOME/gopath/src/github.com/google/trillian
     fi
+  - mkdir protoc
+  - cd protoc
```

Comment:
Suggestion: use a subshell, so we don't change the PWD here.

  - |
    (
      cd protoc
      wget ...
      unzip ...
    )